wolfssl/wolfcrypt/ecc.h

```diff
@@ -632,15 +635,23 @@ int wc_ecc_export_private_raw(ecc_key* key, byte* qx, word32* qxLen,
 #endif /* HAVE_ECC_KEY_EXPORT */
 
 #ifdef HAVE_ECC_KEY_EXPORT
-
+WOLFSSL_API
+int wc_ecc_export_point_der_ex(const int curve_idx, ecc_point* point, byte* out,
+                               word32* outLen, int compressed);
 WOLFSSL_API
 int wc_ecc_export_point_der(const int curve_idx, ecc_point* point,
                             byte* out, word32* outLen);
+WOLFSSL_API
+int wc_ecc_export_point_der_compressed(const int curve_idx, ecc_point* point,
```

Comment:
I would propose making `wc_ecc_export_point_der_compressed` a `WOLFSSL_LOCAL` API. What do you think?

Reply:
Sure, having it as local is okay at this point, or having the _ex one local. The comment was mainly from a concern that if we add too many public api's that do close to the same things, that it may make it harder for users to pick which one they should call.